engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -62,6 +66,7 @@
 
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(LoggerUtil.class);
 
+  private static boolean setMDCProperties = true;
```

Comment:
It is necessary in order to avoid the unnecessary reseting of the same static MDC properties for each appender.